floodviz/views.py

```diff
@@ -1,13 +1,32 @@
 import json
+
 from flask import render_template
 
 from . import app
 from . import hydrograph_utils
 from . import map_utils
 
 
+@app.route('/')
+def root():
+    _hydrograph_helper()
+    mapinfo = _map_helper()
+    return render_template('index.html', mapinfo=mapinfo)
+
+
 @app.route('/home/')
 def home():
+    _hydrograph_helper()
+    return render_template('hydrograph.html')
+
+
+@app.route('/map/')
+def sitemap():
```

Comment:
This will help accomplish the goal of having embeddable web visuals so great idea!